app/Http/Controllers/UsersController.php

```diff
@@ -19,7 +19,20 @@ public function index($id = null) {
             return $this->show($id);
         }
     }
-
+    public function uploadImage(Request $request) {
+        if($request->file('file')!==null) :
+            $image = $request->file('file');
+            $imageName = $image->getClientOriginalName();
+            $destinationPath = public_path('/image');
+            $width = 426;
```

Comment:
You should define these values as const. Then readability and maintainability can increase.